com.github.rajsolai.response.yml

```diff
@@ -0,0 +1,24 @@
+app-id: com.github.rajsolai.response
+
+runtime: org.gnome.Platform
+runtime-version: '3.38'
```

Comment:
```suggestion
runtime-version: '40'
```
maybe?

Reply:
Am not sure about migrating to gnome 40.. May upcoming releases.

Reply:
why not now? it should just work.

Reply:
We're generally not accepting apps using old runtimes unless there's a good reason for it.

Reply:
I tested this using:
> flatpak run --runtime-version=40 com.github.rajsolai.response

and everything seems to work fine. Other than this change everything looks good to merge for me.